docs/marxan/data-computations.md

```diff
@@ -0,0 +1,148 @@
+# Cloud platform computations
+
+- Document created on: 27 September 2021.
+- Latest updated on: 27 September 2021.
+
+Throughout the lifecycle of a Marxan Cloud project, the platform carries out
+tabular or geospatial computations at key steps.
+
+This document summarises which kinds of computations are performed, and outlines
+areas for improvement.
+
+## Planning unit grid
+
+When users choose to use a generated planning unit grid (rather than uploading a
+shapefile with a project-specific grid), planning unit geometries are generated
+via PostGIS.
+
+The time complexity of this operation is generally `O(n)`.
+
+## Protected area of planning units
```

Comment:
(not related to this line but `projects` in general)
Shouldn't we include custom planning area (shapefile upload) ?